standalone/pruntime/enclave/src/lib.rs

```diff
@@ -1634,6 +1637,56 @@ fn test_ink(_input: &Map<String, Value>) -> Result<Value, Value> {
     Ok(json!({}))
 }
 
+const INTERVAL: u64 = 30;
+const MAX_NUM: u128 = 65536*1024;
+
+fn is_prime(num: u128) -> bool {
+    let tmp = num - 1;
+    for i in tmp..=2 {
+        if num % i == 0 {
+            return false;
+        }
+    }
+
+    true
+}
+
+fn check_prime() {
+    for i in 2..MAX_NUM {
+        let _ = is_prime(i);
+    }
+}
+
+fn test_thread(input: &Map<String, Value>) -> Result<Value, Value> {
+    let index = input.get("index").unwrap().as_i64().unwrap();
+
+    println!("=======Begin Thread {:?} Test=======", index);
+
+    loop {
+        let mut counter: u64 = 0;
+
+        let now = SystemTime::now();
+
+        let mut timeout = false;
+
+        while !timeout {
+            check_prime();
+
+            counter += 1;
+
+            if let Ok(elapsed) = now.elapsed() {
```

Comment:
The loop will hard to break if the system time changed backward, due to a time calibration for example.